<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adblock</class>
 <widget class="QWidget" name="adblock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>507</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkEnableAdblock">
     <property name="text">
      <string>&amp;Enable Ad Block</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkHideAds">
     <property name="text">
      <string>&amp;Hide filtered elements</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Automatic Filters</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="addFilters">
         <property name="text">
          <string>Add Filters subscriptions...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="MessageViewer::AdBlockListWidget" name="automaticFiltersListWidget">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="removeSubscription">
         <property name="text">
          <string>Remove subscription</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>301</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Update enabled automatic filters every:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KIntSpinBox" name="spinBox">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="showList">
         <property name="text">
          <string>Show List...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Manual Filters</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="KLineEdit" name="addFilterLineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="insertButton">
           <property name="toolTip">
            <string>Add filter expression</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>252</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="hintLabel">
         <property name="text">
          <string comment="KDE::DoNotExtract">TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="5">
        <widget class="KListWidget" name="manualFiltersListWidget">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="importFilters">
         <property name="text">
          <string>Import Filters...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="exportFilters">
         <property name="text">
          <string>Export filters...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KListWidgetSearchLine" name="searchLine"/>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="editFilter">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="removeButton">
         <property name="toolTip">
          <string>Remove filter expression</string>
         </property>
         <property name="text">
          <string>Remove Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MessageViewer::AdBlockListWidget</class>
   <extends>QListWidget</extends>
   <header>messageviewer/adblock/adblocklistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KListWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>klistwidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
